Sources/ViewInspector/BaseTypes.swift

```diff
@@ -7,38 +7,6 @@ public protocol Inspectable {
     func extractContent(environmentObjects: [AnyObject]) throws -> Any
 }
 
-@available(iOS 13.0, macOS 10.15, tvOS 13.0, *)
```

Comment:
In the `Move code without changes` commit I consolidate the extraction code into a single file.